mask/eolearn/mask/utils.py

```diff
@@ -74,31 +74,37 @@ def resize_images(
     height_width_axis = {2: (0, 1), 3: (0, 1), 4: (1, 2)}
 
     # Old height and width
-    old_size = [data.shape[axis] for axis in height_width_axis[ndims]]
+    old_size = tuple(data.shape[axis] for axis in height_width_axis[ndims])
 
     if new_size is not None and scale_factors is None:
-        scale_factors = [new / old for old, new in zip(old_size, new_size)]
+        scale_factors = tuple(new / old for old, new in zip(old_size, new_size))  # type: ignore[assignment]
     elif scale_factors is not None and new_size is None:
-        new_size = [int(size * factor) for size, factor in zip(old_size, scale_factors)]
+        new_size = tuple()  # type: ignore[assignment]
+        for size, factor in zip(old_size, scale_factors):
+            new_size = (
+                *new_size,
+                int(size * factor),
+            )  # type: ignore[misc, assignment]
```

Comment:
these are tuples, you can do `new_size += (int(size*factor),)`

On a more general note, you have 6 ignores in a single function. It is also deprecated. I don't see a reason to have it typed at that point.

Reply:
untyped, tuple reverted inline